docs/google_analytics_setup.md

```diff
@@ -1,21 +1,19 @@
 # Set up Google Analytics credentials in development
 
-To be able to import data from Google Analytics, a few environment variables need to be set.
-
-1) Create files `config/local_env.yml`
+To be able to import data from Google Analytics, a few environment variables need to be set. If you have not worked in this app before locally, create the file `config/local_env.yml`. This is used to create the environment variables required to connect to, and query the Google API for Google Analytics.
```

Comment:
Some general tech writing style tips which you can use to improve your changes here:

- avoid using the passive voice
- omit non-essential information
- as a rule of thumb, use a new paragraph for anything that's a distinct step or point (if you're not already using a numbered list)

For example:

```suggestion
To import data from Google Analytics, you must set some environment variables.

Create the file `config/local_env.yml` if it does not already exist. This file will contain the environment variables for Google Analytics.
```

Similarly throughout.

Reply:
Thanks @sengi that's very helpful! 😄